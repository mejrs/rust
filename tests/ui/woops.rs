#![crate_type = "lib"]
#![feature(rustc_attrs)]

#[rustc_as_ptr] //~ERROR
pub struct X;
