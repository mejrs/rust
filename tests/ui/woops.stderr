error: `#[rustc_as_ptr]` is not valid at that position. It is only allowed in ["function", "method"]
  --> $DIR/woops.rs:4:1
   |
LL | #[rustc_as_ptr]
   | ^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

